cosmos/operators/docker.py

```diff
@@ -20,7 +20,7 @@
     )
 
 
-class DbtDockerBaseOperator(DockerOperator, DbtBaseOperator):
+class DbtDockerBaseOperator(DockerOperator, DbtBaseOperator):  # type: ignore[misc] # ignores subclass MyPy error
```

Comment:
Which error...? Is there anything we could do to improve this?

Reply:
Resolving in a separate, future ticket.